Core/Core/Assignments/AssignmentDueDates/View/AssignmentDueDateItemView.swift

```diff
@@ -0,0 +1,95 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2023-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import SwiftUI
+
+struct AssignmentDueDateItemView: View {
+    private var model: AssignmentDueDateItemViewModel
+
+    public init(model: AssignmentDueDateItemViewModel) {
+        self.model = model
+    }
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: 0) {
+            Text(model.title)
+                .font(.heavy24).foregroundColor(.textDarkest)
+                .padding(16)
+            Section(
+                label: Text("For"),
+                content: { Text(model.assignee) }
+            )
+            Section(
+                label: Text("Available From"),
+                content: {
+                    if let fromEmptyAccessibility = model.fromEmptyAccessibility {
```

Comment:
I wonder if we could have a universal text type for handling placeholders.

Reply:
it's not that common, to have a different accessibility for a placeholder, so I'll leave this, the next time it comes up, we can think of something.